barcode/site_sections/barcode.py

```diff
@@ -0,0 +1,12 @@
+from barcode import *
+from barcode_utils import generate_barcode_from_badge_num, get_badge_num_from_barcode
+
+# TODO: restrict to just admins for this stuff, probably.
+@all_renderable(c.PEOPLE)
+class Root:
+    def test(self):
+        badge_num = 3
+        encrypted = generate_barcode_from_badge_num(badge_num=badge_num)
+        decrypted = get_badge_num_from_barcode(barcode_num=encrypted)
+
+        return "badge_num = " + str(decrypted['badge_num']) + ", event_id = " + str(decrypted['event_id'])
```

Comment:
test only.  just generating a barcode from a badge#, and then generating the same badge# from the barcode